### DISCUSSION TITLE
Is it possible to integrate email magic link authentication in web extension

### DISCUSSION THREAD

**Discussion by gypsicoder** [supabase/supabase#12591] [Questions]
Can we integrate email magic link authentication in web extensions? In that case what configurations we have to do?
I was trying to put the chrome extension popup page url in the redirect url from where I was initiating the request

``
const { data, error } = await supabaseClient.auth.signInWithOtp({email,
      options: {
        emailRedirectTo:
          "chrome-extension://pbfpfhnegincokdincpdpdaaplnojlog/popup.html"
      }
    })
``

But, it's not working.

**Comment by rafaelfcsouza** [comment]
Having an issue with this too. I'm getting 
`error=unauthorized_client&error_code=401&error_description=Email+link+is+invalid+or+has+expired` even on first use of the email link
